### DISCUSSION TITLE
BookLibConnect won't start

### DISCUSSION THREAD

**Discussion by JammyDonut** [audiamus/BookLibConnect#89] [General]
hi, 
I'm trying to install for the first time (no previous install), using installer BookLibConnect-0.11-Setup.exe.
Install goes fine, but when starting the program, it appears briefly in Task Manager and then disappears.   
I'm using Windows 10 22H2.

I'm not sure where the application log is , but in the Event Viewer, there's some errors shown.

> Application: BookLibConnect.exe
> CoreCLR Version: 6.0.121.56705
> .NET Version: 6.0.1
> Description: The process was terminated due to an unhandled exception.
> Exception Info: System.IO.FileLoadException: Could not load file or assembly 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken= _(removed, is this sensitive??)_ '. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
> File name: 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=_(removed)_'
>    at core.audiamus.connect.app.gui.Program.Main(String[] args)

and

> Faulting application name: BookLibConnect.exe, version: 0.11.0.0, time stamp: 0x63558578
> Faulting module name: KERNELBASE.dll, version: 10.0.19041.2364, time stamp: 0x5b7d4d22
> Exception code: 0xe0434352
> Fault offset: 0x000000000002cd29
> Faulting process ID: 0x4780
> Faulting application start time: 0x01d937f3ddf14b86
> Faulting application path: C:\Program Files\audiamus\Book Lib Connect\BookLibConnect.exe
> Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
> Report ID: 737aaffd-b826-4015-99bb-a6c60ce385e3
> Faulting package full name: 
> Faulting package-relative application ID: 

any ideas? 

thanks

**Comment by audiamus** [comment]
It looks like some CLR version mismatch but I don't really know. You could try to update to the latest 6.0 runtime (6.0.13):
https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.13-windows-x64-installer

**Reply by JammyDonut** [reply]
Thanks for that! it's now working

 - turns out I had a very old dotnet runtime (  dotnet --info   was showing 6.0.1  )
